### ISSUE TITLE
Subtitles get downloaded with non english names

### ISSUE THREAD

**Issue by Winor** [croneter/PlexKodiConnect#394]
Hey, I'm running OpenELEC 8.0.4 on my RPi and Hebrew subtitles won't show up although being discovered.
I saw issues #378 and #342 but I'm using PKC 1.8.18 and still getting this:

```
12:41:03.053 T:1451226016   DEBUG: PLEX.PlexAPI: Writing temp subtitle to /storage/.kodi/userdata/addon_data/plugin.video.plexkodiconnect/temp/subtitle.עברית.srt
12:41:03.055 T:1451226016   DEBUG: PLEX.PlexAPI: Need to slugify the filename /storage/.kodi/userdata/addon_data/plugin.video.plexkodiconnect/temp/subtitle.עברית.srt
12:41:03.130 T:1451226016   DEBUG: PLEX.PlexAPI: Writing temp subtitle to /storage/.kodi/userdata/addon_data/plugin.video.plexkodiconnect/temp/subtitle.English.srt
12:41:03.186 T:1451226016   DEBUG: PLEX.PlexAPI: Writing temp subtitle to /storage/.kodi/userdata/addon_data/plugin.video.plexkodiconnect/temp/subtitle.עברית.srt
12:41:03.189 T:1451226016   DEBUG: PLEX.PlexAPI: Need to slugify the filename /storage/.kodi/userdata/addon_data/plugin.video.plexkodiconnect/temp/subtitle.עברית.srt
```

See full [kodi.log](https://github.com/croneter/PlexKodiConnect/files/1667948/kodi.2.log) file.

I tried with more videos but I'm getting the same issue.
When using PKC on Windows everything works as expected.
Any idea what's going on here?

**Comment by Pavuucek** [comment]
I've looked at your log and you're also getting an awful lot of errors like these:
`12:00:37.484 T:1730904992   ERROR: GetCachedTexture, failed on url '/storage/.kodi/addons/script.plex/icon.png'
12:00:37.769 T:1730904992   ERROR: SQL: [Textures13.db] The database disk image is malformed
                                            Query: DELETE FROM texture WHERE url='/storage/.kodi/addons/script.plex/icon.png'
12:00:37.769 T:1730904992   ERROR: AddCachedTexture failed on url '/storage/.kodi/addons/script.plex/icon.png'`
Which suggests there is a problem with your SD card. You might need to get a new one. Meanwhile try to reinstall PKC and see if that helps.

Also please [RESET YOUR TOKEN!](http://support.plex.tv/articles/204059436-finding-an-authentication-token-x-plex-token) and see [here](https://github.com/croneter/PlexKodiConnect/wiki/How-to-Report-A-Bug) before posting logs next time :-)

**Comment by Winor** [comment]
Thanks for your response and for letting me know that I just shared my token with everyone 😳
I will now replace the SD card, reinstall everything and let you know if that helped :D

**Comment by Winor** [comment]
Okay, so I reinstalled everything on a new SD card and now some of the videos get subtitles downloaded as intended with the 3 letters languageCode in the filename while other videos get the language in the filename like in my first post and it's seems like it now happens with specific videos although I'm not sure why.

Note that the video I tested in my first post is the same video that now works.
also on my Windows machine all of my videos get the 3 letters languageCode filename and work perfectly.

Here is the new [kodi.log](https://github.com/croneter/PlexKodiConnect/files/1671479/kodi.3.log) file from my new OpenELEC installation.
What should I try next?
